manifests/a/AntiMicroX/antimicrox/3.3.3/AntiMicroX.antimicrox.locale.en-US.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: AntiMicroX.antimicrox
+PackageVersion: 3.3.3
+PackageLocale: en-US
+Publisher: AntiMicroX

```

```suggestion
Publisher: Humanity
```
This should match Control Panel